cmd/dockerd/daemon_test.go

```diff
@@ -120,7 +120,7 @@ func TestLoadDaemonCliConfigWithTLSVerify(t *testing.T) {
 	}
 
 	if !loadedConfig.TLS {
-		t.Fatalf("expected TLS enabled, got %q", loadedConfig)
+		t.Fatalf("expected TLS disabled, got %q", loadedConfig)
```

Comment:
But this fatal is shown if `loadedConfig.TLS` is false? Seems to me that the message is correct here; "the tests expects that TLS is enabled (true), but it is false"